commands/aliases.go

```diff
@@ -96,44 +110,41 @@ func aliasCommand(ctx context.Context, args []string) (string, error) {
 		p := dgwidgets.NewPaginator(
 			ctx.Value(messageContextValue("Session")).(*discordgo.Session),
 			ctx.Value(messageContextValue("ChannelID")).(string))
+
+		var sortedAliases []string
+		for a := range aliasMap {
+			sortedAliases = append(sortedAliases, a)
+		}
+		sort.Strings(sortedAliases)
+
+		var aliasListing []string
+		for i, a := range sortedAliases {
+			aliasListing = append(aliasListing, fmt.Sprintf("%d) **%s**", i+2, a))
```

Comment:
Jus sayin, we could reuse the sortedAliases slice here

Reply:
Sorted aliases has to be used below as the original strings because they need to be the keys to the `aliasMap` map. 
```golang
for _, aliasName := range sortedAliases {
	info := aliasMap[aliasName]
	embed := &discordgo.MessageEmbed{
		Title:       aliasName,
		Description: info.Value,
	}

	// if the link is to an image, we add the image itself to the embed
	kind := info.Kind
	if kind == kindIMAGE {
		embed.Image = &discordgo.MessageEmbedImage{
			URL: info.Value,
		}
	}
	p.Add(embed)
}```

The individual command pages must be added in the same order as the first page lists them as.

Reply:
Ah yea i getcha. Cant fully review this but i see what you mean, im sure you know whats going on :+1: